### PR TITLE
0.10.2 documentation update

### PR DESCRIPTION
**Pull request by warp-core**
**Documentation**

## Summary
Updates the changelog with all the changes since 0.10.1 (so far).
Adds three new names to the list of contributors in the credits.
Also updates versions numbers and adds an entry to the appdata.xml file for 0.10.2.

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+      * New civilian ship hails. (@LazerLit)
+      * New human news items. (@dorbarker)
+      * New jobs in Free Worlds space during the war that involve Military cargo. (@dorbarker)
+    * Mission changes:
+      * The "FW Pug 2C" missions now use autoconditions to ensure the appropriate one is offered instead of relying on MissionAction blocking. (@warp_core)
+      * The "Street Swindle" mission is now invisible, and uses conversation actions to take credits from the player and is always declined. (@lumbar527)
+      * Small wording change to "Earth Retirement" mission. (@lumbar527)
+      * Various wording changes to Gegno intro missions. (@Saugia)
+      * The missions that trigger the events for the Kestrel to be made available will now fail immediately instead of staying active indefinitely doing nothing. (@warp-core)
+      * Added a branch for "FW Senate 1B" for if hte player already has a jump drive installed. (@lumbar527)
```
Comment (lumbar527) [inline changelog:25]:
```suggestion
      * Added a branch for "FW Senate 1B" for if the player already has a jump drive installed. (@lumbar527)
```

File: changelog [hunk 2]
```diff
@@ -1,3 +1,47 @@
+Version 0.10.2
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@nothing-but-the-rain, @warp-core, @waterhouse)
+      * "Pact Recon 0" can no longer be offered from Glaze. (@UnorderedSigh)
+      * "Hai Reveal: Early Arrival: Alondo" and "Hai Reveal: Late Arrival: Alondo" can no longer offer from Hai-home. (@warp-core)
+      * Text in "Liberate Kornephoros" now correctly refers to Carriers as the attacking ships, not Cruisers. (@tibetiroka)
```
Comment (lumbar527) [inline changelog:7]: Should this be with `mission changes` with other wording changes?

**Review by lumbar527 — APPROVED** [review]
Mostly good? (I don't really know changelog style stuff)

Yay I'm a contributor!

File: changelog (hunk 2 above)
Comment (warp-core) [reply changelog:7]: I put it here because the affected text was incorrect prior to this change.

**Review by Amazinite — APPROVED** [review]

**Review by quyykk — APPROVED** [review]

**Pull request merged**